shipit/trunk/shipit.php

```diff
@@ -605,20 +605,29 @@ function createShipment($company, $skus, $sellerSetting, $insuranceSetting, $ord
         $measure->getMeasure(),
         $insurance->getInsurance());
       // MAKE API CALL
-      if ($sellerSetting->automatic_delivery === false) {
-        $integration = new Integration(get_option('shipit_user')['shipit_user'], get_option('shipit_user')['shipit_token']);
-        return $integration->massiveOrders(['orders' => [$orderPayload->build()]]);
-      } elseif ($shipit) {
-          $order = new WC_Order($order->get_id());
-          $order->add_order_note('Ya que aplicaron el metodo de retiro en tienda.');
+      $same_day = Courier::sameDayCourier($courier_name, get_option('shipit_user')['shipit_user'], get_option('shipit_user')['shipit_token'], 'v4');
+      if ($same_day) {
+        sendOrderOrShipment($sellerSetting->same_day->automatic_delivery, $orderPayload, $shipit, $order);
       } else {
-          $core = new Core(get_option('shipit_user')['shipit_user'], get_option('shipit_user')['shipit_token'], 'v4');
-          return $core->shipments(['shipment' => $orderPayload->build()]);
+        sendOrderOrShipment($sellerSetting->automatic_delivery, $orderPayload, $shipit, $order);
       }
     }
   }
 }
 
+function sendOrderOrShipment($automatic_delivery, $orderPayload, $shipit, $order) {
+  if ( $automatic_delivery === false ) {
+    $integration = new Integration(get_option('shipit_user')['shipit_user'], get_option('shipit_user')['shipit_token']);
+    return $integration->massiveOrders(['orders' => [$orderPayload->build()]]);
+  } elseif ($shipit) {
+      $order = new WC_Order($order->get_id());
+      $order->add_order_note('Ya que aplicaron el metodo de retiro en tienda.');
```

Comment:
I don't understand this message, maybe be
```suggestion
      $order->add_order_note('Ya fue aplicado el método de retiro en tienda.');
```

Reply:
yes, it is because it is the continuation of another message

Reply:
Perfect, but 'método'